NEWS.rst

```diff
@@ -75,6 +75,9 @@ attributes of the ``SeqFeature`` which was done without a deprecation period.
 They are again aliases for ``.location.strand`` etc, but trigger deprecation
 warnings.
 
+The unused ``version`` parameter to the ``weblogo`` method of
+``Bio.motifs.Motif`` is also now deprecated.
+
```

Comment:
I think this is too minor for the NEWS file - in general things like this only go in the DEPRECATED file.

Reply:
OK, I just pushed another commit that removes the NEWS entry.